add a phrase about the contract id (#233)
digital-asset/docs.daml.com: docs/2.6.1/docs/daml/reference/contract-keys.rst

Contract keys are an optional addition to templates. They let you specify a way of uniquely identifying contracts, using the parameters to the template - similar to a primary key for a database.

> You can use contract keys to stably refer to a contract, even through iterations of instances of it in which the contract id changes.

Reviewer: ```suggestion
Contract keys do not change and can be used to refer to a contract even when the contract id changes.
```
I think this simplifies it even more (and also gets rid of the adverb).
